en/index.html

```diff
@@ -20,12 +20,12 @@
   <main id="content">
     <article>
       <section>
-        <h1 class="subject-main">Dmitry Bogatov has spent <strong>106 days</strong> in pretrial detention in Moscow. He's now waiting for trial under house arrest.</h1>
+        <h1 class="subject-main">Dmitry Bogatov has spent <strong>106 days</strong> in pretrial detention in Moscow. He's now waiting for trial under house arrest</h1>
```

Comment:
Wait, if you have dot after first sentences, why do you remove it after the second one.